xapian-maintainer-tools/xapian-check-patch

```diff
@@ -152,7 +153,13 @@ while (<>) {
 	$check_space_tab = 1;
 	$check_trailing = 1;
 	$doxygen_first_line = 1;
-	if ($ext eq 'cc') {
+        $check_end_new_line = 1;
+        if  ($fnm =~ m!xapian-applications/omega/testfiles!) {
+	    $check_space_tab = 0;
+	    $check_trailing = 0;
+	    $doxygen_first_line = 0;
+	    $check_end_new_line = 0;
+        } elsif ($ext eq 'cc') {
```

Comment:
I've cherry-picked this commit to master as dca1a763b4ca5a0185cbd3a1d17adb44ceca440b (with some whitespace fixes squashed in - there's a double space after `if` above, and some of the indentation in this hunk doesn't use tabs.)